projects/packages/waf/tests/php/functions-wordpress.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
I've not seen anything like this before, maybe you can explain it to me at the meetup?

Reply:
I'm glad to do that! But to put it in a nutshell, this is what I meant by 

> I took the get_option mock for the tests from the jetpack-autoloader package, this seemed like the most reasonable solution.

Basically, these [already exist here](https://github.com/Automattic/jetpack/blob/master/projects/packages/autoloader/tests/php/lib/functions-wordpress.php), and I copied parts to be able to mock that WordPress-internal `get_option()` function (which is not known by the tests!).

Reply:
P.S.: I now also did the same thing for `trailingslashit()` to be available in the tests.

Reply:
@leogermani Is this something where we could rely on `WorDBless` for some of those things?

Reply:
@jwidavid I'd say its something to keep in the back of our heads. Since it would only help for one single mock function in there currently, I would argue that the benefits of adding that extra package do not outweigh the "costs" (yet), and by costs I mean the maintenance that comes involved with it, you'll have to upgrade the dependency at appropriate times and account for any changes made and such.

I'm certainly not against adding it in the future and its a good thing to know that it exists.